doc/source/tutorial/stats/rv_infrastructure.md

```diff
@@ -0,0 +1,772 @@
+
+```{code-cell} ipython3
+import matplotlib.pyplot as plt
+x = np.linspace(2, 10, 300)
+
+c, loc, scale = res.x
+X = Weibull(c=c)*scale + loc
+plt.plot(x, X.pdf(x), '-', label='numerical optimization')
+
+c, loc, scale = params
+Y = Weibull(c=c)*scale + loc
+plt.plot(x, Y.pdf(x), '--', label='`scipy.stats.fit`')
+
+plt.hist(data, bins=np.linspace(2, 10, 30), density=True, alpha=0.1)
+plt.xlabel('x')
+plt.ylabel('pdf(x)')
+plt.legend()
+```
+
+## New Features
+
++++
+
+<a id='Transformations'></a>
+### Transformations
+
+Mathematical transformations can be applied to random variables. For instance, many elementary arithmetic operations (`+`, `-`, `*`, `/`, `**`) between random variables and scalars work.
+
+For instance, we can see that the reciprocal of a Weibull-distributed RV is distributed according to `scipy.stats.invweibull`. (An "inverse" distribution is typically the distribution underlying the reciprocal of a random variable.)
+
+```{code-cell} ipython3
+c = 10.6
+
+X = Weibull(c=10.6)  
+Y = 1 / X  # compare to `invweibull`
+Y.plot();
+
+x = np.linspace(0.8, 2.05, 300)
+plt.plot(x, stats.invweibull(c=c).pdf(x), '--')
+```
+
+`scipy.stats.chis2` describes a sum of the squares of normally-distributed random variables.
+
+```{code-cell} ipython3
+X = stats.Normal()
+Y = X**2  # compare to chi2
+Y.plot(t=('x', 0, 5));
+
+x = np.linspace(0, 5, 300)
+plt.plot(x, stats.chi2(df=1).pdf(x), '--')
+```
+
+`scipy.stats.foldcauchy` describes the absolute value of a Cauchy-distributed random variable. (A "folded" distribution is the distribution underlying the absolute value of a random variable.)
+
+```{code-cell} ipython3
+Cauchy = stats.make_distribution(stats.cauchy)
+c = 4.72
+
+X = Cauchy() + c  
+Y = abs(X)  # compare to `foldcauchy`
+Y.plot(t=('x', 0, 60));
+
+x = np.linspace(0, 60, 300)
+plt.plot(x, stats.foldcauchy(c=c).pdf(x), '--')
+```
+
+`scipy.stats.lognormal` describes the exponential of a normally distributed random variable. It is so-named because the logarithm of the resulting random variable is normally distributed. (In general, a "log" distribution is typically the distribution underlying the *exponential* of a random variable.)
+
+```{code-cell} ipython3
+u, s = 1, 0.5
+
+X = stats.Normal()*s + u
+Y = stats.exp(X)  # compare to `lognorm`
+Y.plot(t=('x', 0, 9));
+
+x = np.linspace(0, 9, 300)
+plt.plot(x, stats.lognorm(s=s, scale=np.exp(u)).pdf(x), '--')
+```
+
+`scipy.stats.loggamma` describes the logarithm of of a Gamma-distributed random variable. Note that the more common name of this distribution would be [exp-gamma](https://reference.wolfram.com/language/ref/ExpGammaDistribution.html.en#:~:text=The%20term%20exp%2Dgamma%20is,of%20qualitatively%20similar%20probability%20distributions.) because the exponential of the RV is gamma-distributed.
+
+```{code-cell} ipython3
+Gamma = stats.make_distribution(stats.gamma)
```

Comment:
What's the rationale for having `X = stats.Normal()` but not `X = stats.Gamma()`?

Reply:
For the _first_ distribution with a new class, `Normal` seemed appropriate. It's pretty fundamental.

Personally, I've envisioned adding a few of the most fundamental distributions back with the new naming scheme because the performance, accuracy, and interface can be a bit better than if we rely on `make_distribution` for them. For instance, `Uniform` can be parametrized by the endpoints rather than location/scale, and new options for private methods can improve the speed and accuracy of the public ones. As for why that's not *all* done yet, well, it's mostly a matter of time. `make_distribution` barely made it into 1.15, and that seemed more important to me, so that's where I spent my time.

Reply:
> As for why that's not all done yet, well, it's mostly a matter of time. make_distribution barely made it into 1.15, and that seemed more important to me, so that's where I spent my time.

Completely understandable. Might still be worth a thought to note that the new infrastructure will still receive updates & improvements in the . The text kinda reads like it's in its final state already.

Reply:
Hah, that would be nice. Please see if the (new) conclusion begins to address this point, and let me know if there are particular places where the language might be changed to avoid the appearance that we think this is all done.